docs/automated-updates.md

```diff
@@ -0,0 +1,147 @@
+# Automated Template Updates with GitHub Copilot
+
+This guide provides fully automated methods to validate and update your ioBroker Copilot instructions template using GitHub Copilot, ensuring all custom sections are preserved.
```

Comment:
duplication should be avoided, instructions already available in the template can be removed from [custom section], update related comments, instructions etc accordingly